### DISCUSSION TITLE
Edge Function ignores apikey and allows access without key

### DISCUSSION THREAD

**Discussion by wozdog-oz** [sqlitecloud/discussions#28] [Bug Report] [answered]
I created an edge function and selected and APIKEY in the function definition UI. After deploying the function, I am able to execute the function without specifying the API key. So the function is callable by anyone. This seems to be be a serious bug. 

Is there any way to ensure that the functions require the apikey?

**Comment by andinux** [comment, marked as answer]
Please see the latest reply in https://github.com/orgs/sqlitecloud/discussions/27, let me know if you require any further assistance

**Reply by wozdog-oz** [reply]
Ok, not a bug!  The problem was between the chair and the keyboard. :-) 
thanks
